pytest_mypy/item.py

```diff
@@ -163,6 +167,11 @@ def typecheck_in_new_subprocess(self, execution_path: Path, mypy_cmd_options: Li
 
         # add current directory to path
         self.environment_variables['PYTHONPATH'] = str(execution_path)
+        try:
+            # Windows requires this to be set, otherwise the interpreter crashes
+            self.environment_variables["SYSTEMROOT"] = os.environ["SYSTEMROOT"]
+        except KeyError:
```

Comment:
Can we refactor this not to contain `except/pass`?

Reply:
Sure! Although, I'd be interested to get your thoughts on why.